apps/app-article-server/src/browser/components/article.js

```diff
@@ -32,6 +32,22 @@ class Article extends Component {
       document.getElementsByTagName("HTML")[0].setAttribute("data-theme", "dark");
     }
     this.getMostReadArticles();
+    this.handleMessage = this.handleMessage.bind(this);
```

Comment:
I think it would have been fine even in the first place, with just removing this line. JS has no limit about having functions as record values even without binds, what the bind does is that is that it sets the `this` value inside them point to the given value.